libs/langgraph/langgraph/graph/state.py

```diff
@@ -86,7 +86,18 @@ def _get_node_name(node: RunnableLike) -> str:
 
 @dataclasses.dataclass(**_DC_KWARGS)
 class GraphCommand(Generic[N], Command[N]):
-    """One or more commands to update a StateGraph's state and go to, or send messages to nodes."""
+    """One or more commands to update a StateGraph's state and go to, or send messages to nodes.
+
+    Args:
+        goto: name of the node to navigate to next.
+            If not specified, the graph will halt after executing the current superstep.
+        graph: graph to send the command to. Supported values are:
```

Comment:
`graph` feels like this should be the first parameter so it's clear that the rest of the things are sent to this graph